chapters/00.Introduction/00-GetGoingWithJulia.md

```diff
@@ -0,0 +1,153 @@
+---
+author: dimiboeckaerts
+---
+
+# Let's get going with Julia
+
+Let's get going with Julia! This summary-based guide into programming with Julia is meant for students and researchers with a general background in programming but no specific knowledge of Julia. The first two sections get you up and running in Julia, while the third and fourth section dive into some more (bio)engineering-oriented material. Finally, the last section provides additional resources to explore more from Julia. Happy learning!
+
+[TOC]
+
+
+## The basics
+
+What is Julia again?
+https://julialang.org/learning/getting-started/
+
+Why should you learn Julia?
+https://blog.goodaudience.com/10-reasons-why-you-should-learn-julia-d786ac29c6ca
+
+Let's download & install Julia. You can either download the language itself and engage with it through the command-line (or notebooks, see below):
+https://julialang.org/downloads/
+or you can download JuliaPro (requires a free account) which comes with Juno, an IDE to work in:
+https://juliacomputing.com/products/juliapro.html
+
+First baby steps around the block:
+https://docs.julialang.org/en/v1/manual/getting-started/
+
+Learn the basics of Julia:
+https://juliabyexample.helpmanual.io/
+https://learnxinyminutes.com/docs/julia/
+
+Learn more about types, multiple dispatch and structs:
+https://towardsdatascience.com/how-to-learn-julia-when-you-already-know-python-641ed02b3fa7
+
+
+## Notebooks & Visualization
+
+### Jupyter notebook & IJulia
+Coupling Julia to Jupyter notebooks is done with IJulia:
+https://www.youtube.com/watch?v=oyx8M1yoboY
+
+### Pluto
+Pluto is a lightweight, reactive notebook for Julia, with a lot of useful features:
```

Comment:
Pluto will be used for the projects while we use Jupyter notebook for the exercises.